profiler/src/Tools/ETW/dd-prof-etw/dd-prof-etw-client/EtwEventDumper.h

```diff
@@ -4,11 +4,15 @@
 
 #include "..\..\..\..\ProfilerEngine\Datadog.Profiler.Native.Windows\ETW\IEtwEventsReceiver.h"
 
+#include <memory>
 #include <string>
 
 class EtwEventDumper : public IEtwEventsReceiver
 {
 public:
+    EtwEventDumper();
+    ~EtwEventDumper();
```

Comment:
why declaring those ctor/dtor ? compiler will do it just fine for you

Reply:
✅  done